### PR TITLE
Add Gesture Extensions Bindings

### PR DESCRIPTION
**Pull request by brminnick**
This PR adds documentation for Gesture Extensions in C# Markup: https://github.com/CommunityToolkit/Maui.Markup/pull/192

### CODE REVIEW

File: docs/maui/markup/extensions/bindable-object-extensions.md [hunk 1]
```diff
@@ -118,6 +118,47 @@ new Button()
...
+        numberOfClicksRequired: 2));
+```
+
+### BindSwipeGesture
+
+The following example demonstrates how to create a `SwipeGestureRecognizer` that requires `SwipeDirection.Up` for its `SwipeDirection` and a minumum 200-point distance for its `Threshold`, then attach it to a `Label` and bind it to an `ICommand` in our ViewModel:
+
+```cs
+new Label()
+    .BindClickGesture(
```
Comment (bijington) [inline docs/maui/markup/extensions/bindable-object-extensions.md:143]:
```suggestion
    .BindSwipeGesture(
```

File: docs/maui/markup/extensions/bindable-object-extensions.md [hunk 2]
```diff
@@ -118,6 +118,47 @@ new Button()
...
+        threshold: 200);
+```
+
+### BindTapGesture
+
+The following example demonstrates how to create a `ClickGestureRecognizer` that requires `2` taps to activate, attach it to a `Label` and bind it to an `ICommand` in our ViewModel:
+
+```cs
+new Label()
+    .BindClickGesture(
```
Comment (bijington) [inline docs/maui/markup/extensions/bindable-object-extensions.md:156]:
```suggestion
    .BindTapGesture(
```

File: docs/maui/markup/extensions/bindable-object-extensions.md [hunk 3]
```diff
@@ -118,6 +118,47 @@ new Button()
         mode: BindingMode.OneTime);
 ```
 
+## Gesture Binding
+
+Gesture bindings allow us to create an `ClickGestureRecognizer`, `SwipeGestureRecognizer`, `TapGestureRecognizer`, attach it to any element that implements `IGestureRecognizer` and bind it to an `ICommand` in our ViewModel.
+
+### BindClickGesture
+
+The following example demonstrates how to create a `ClickGestureRecognizer` that requires `2` clicks to activate, attach it to a `Label` and bind it to an `ICommand` in our ViewModel:
```
Comment (bijington) [inline docs/maui/markup/extensions/bindable-object-extensions.md:127]:
```suggestion
The following example demonstrates how to create a `ClickGestureRecognizer` that requires `2` clicks to activate, attach it to a `Label` and bind it to an `ICommand` property called _ClickCommand_ in our ViewModel:
```

File: docs/maui/markup/extensions/bindable-object-extensions.md [hunk 4]
```diff
@@ -118,6 +118,47 @@ new Button()
...
+```
+
+### BindSwipeGesture
+
+The following example demonstrates how to create a `SwipeGestureRecognizer` that requires `SwipeDirection.Up` for its `SwipeDirection` and a minumum 200-point distance for its `Threshold`, then attach it to a `Label` and bind it to an `ICommand` in our ViewModel:
+
+```cs
+new Label()
+    .BindClickGesture(
+        static (ViewModel vm) => vm.ClickCommand,
```
Comment (bijington) [inline docs/maui/markup/extensions/bindable-object-extensions.md:144]:
```suggestion
        static (ViewModel vm) => vm.SwipeCommand,
```

File: docs/maui/markup/extensions/bindable-object-extensions.md [hunk 5]
```diff
@@ -118,6 +118,47 @@ new Button()
...
+        commandBindingMode: BindingMode.OneTime,
+        numberOfClicksRequired: 2));
+```
+
+### BindSwipeGesture
+
+The following example demonstrates how to create a `SwipeGestureRecognizer` that requires `SwipeDirection.Up` for its `SwipeDirection` and a minumum 200-point distance for its `Threshold`, then attach it to a `Label` and bind it to an `ICommand` in our ViewModel:
```
Comment (bijington) [inline docs/maui/markup/extensions/bindable-object-extensions.md:139]:
```suggestion
The following example demonstrates how to create a `SwipeGestureRecognizer` that requires `SwipeDirection.Up` for its `SwipeDirection` and a minumum 200-point distance for its `Threshold`, then attach it to a `Label` and bind it to an `ICommand` property called _SwipeCommand_ in our ViewModel:
```

File: docs/maui/markup/extensions/bindable-object-extensions.md [hunk 6]
```diff
@@ -118,6 +118,47 @@ new Button()
...
+```
+
+### BindTapGesture
+
+The following example demonstrates how to create a `ClickGestureRecognizer` that requires `2` taps to activate, attach it to a `Label` and bind it to an `ICommand` in our ViewModel:
+
+```cs
+new Label()
+    .BindClickGesture(
+        static (ViewModel vm) => vm.ClickCommand,
```
Comment (bijington) [inline docs/maui/markup/extensions/bindable-object-extensions.md:157]:
```suggestion
        static (ViewModel vm) => vm.TapCommand,
```

File: docs/maui/markup/extensions/bindable-object-extensions.md [hunk 7]
```diff
@@ -118,6 +118,47 @@ new Button()
...
+        commandBindingMode: BindingMode.OneTime,
+        direction: SwipeDirection.Up,
+        threshold: 200);
+```
+
+### BindTapGesture
+
+The following example demonstrates how to create a `ClickGestureRecognizer` that requires `2` taps to activate, attach it to a `Label` and bind it to an `ICommand` in our ViewModel:
```
Comment (bijington) [inline docs/maui/markup/extensions/bindable-object-extensions.md:152]:
```suggestion
The following example demonstrates how to create a `ClickGestureRecognizer` that requires `2` taps to activate, attach it to a `Label` and bind it to an `ICommand` property called _TapCommand_ in our ViewModel:
```

**Review by bijington — COMMENTED** [review]
Thanks @brminnick I have jut fixed a couple of mistakes and added a little bit extra context

**Comment by bijington** [comment]
Sorry for the noise, I thought I could batch commit those changes but the button was disabled 😢

**Review by bijington — APPROVED** [review]

**Comment by brminnick** [comment]
No worries at all! Thanks Shaun!!

**Pull request merged**